sympy/integrals/tests/test_meijerint.py

```diff
@@ -631,5 +631,14 @@ def test_fresnel():
     assert expand_func(integrate(sin(pi*x**2/2), x)) == fresnels(x)
     assert expand_func(integrate(cos(pi*x**2/2), x)) == fresnelc(x)
 
+
 def test_issue_6860():
     assert meijerint_indefinite(x**x**x, x) is None
+
+
+def test_issue_8368():
+    assert meijerint_indefinite(cosh(x)*exp(-x*t), x) == (
+        -t*exp(2*x)/(2*t**2*exp(x)*exp(t*x) - 2*exp(x)*exp(t*x)) -
+        t/(2*t**2*exp(x)*exp(t*x) - 2*exp(x)*exp(t*x)) -
+        exp(2*x)/(2*t**2*exp(x)*exp(t*x) - 2*exp(x)*exp(t*x)) +
+        1/(2*t**2*exp(x)*exp(t*x) - 2*exp(x)*exp(t*x)))
```

Comment:
I tried something judicious for those returns obtained after rewriting in terms of exp.